src/OctoshiftCLI.Tests/gei/Handlers/MigrateRepoCommandHandlerTests.cs

```diff
@@ -737,10 +747,12 @@ public async Task Ghes_With_NoSslVerify_Uses_NoSsl_Client()
             _mockSourceGithubApi.Setup(x => x.GetArchiveMigrationUrl(SOURCE_ORG, gitArchiveId).Result).Returns(gitArchiveUrl);
             _mockSourceGithubApi.Setup(x => x.GetArchiveMigrationUrl(SOURCE_ORG, metadataArchiveId).Result).Returns(metadataArchiveUrl);
 
-            _mockHttpDownloadService.Setup(x => x.DownloadToBytes(gitArchiveUrl).Result).Returns(gitArchiveContent);
-            _mockHttpDownloadService.Setup(x => x.DownloadToBytes(metadataArchiveUrl).Result).Returns(metadataArchiveContent);
-            _mockAzureApi.Setup(x => x.UploadToBlob(It.IsAny<string>(), gitArchiveContent).Result).Returns(authenticatedGitArchiveUrl);
-            _mockAzureApi.Setup(x => x.UploadToBlob(It.IsAny<string>(), metadataArchiveContent).Result).Returns(authenticatedMetadataArchiveUrl);
+            _mockHttpDownloadService.Setup(x => x.DownloadToFile(gitArchiveUrl, gitArchiveFilePath));
+            _mockHttpDownloadService.Setup(x => x.DownloadToFile(metadataArchiveUrl, metadataArchiveFilePath));
+
+            _mockFileSystemProvider.Setup(x => x.OpenRead(gitArchiveFilePath)).Returns(It.IsAny<FileStream>());
```

Comment:
same

Reply:
Addressed in 7d436e04071bf868990e1d32d6276668c1db1a35 :tada: